### DISCUSSION TITLE
Config webirc to get real hostnames

### DISCUSSION THREAD

**Discussion by nitro212** [inspircd/inspircd#2140] [Support] [answered]
I can’t config to get work webirc to get real hostnames from users between kiwiirc and inspircd 4.

**Comment by sadiepowell** [comment]
Can you share your InspIRCd and KiwiIRC WebIRC config?

**Reply by nitro212** [reply, marked as answer]
I get the work done. Some users in this forum have the same issue and I view to problem! Thanks
